### ISSUE TITLE
How to use Access8Math editor?

### ISSUE THREAD

**Issue by hjy1210** [tsengwoody/Access8Math#85]
Following NVDA tools menu > Access8Math > Editor,  Access8Math Editor appeared.

Access8Math Editor has three menu items with submenus
* File
  * New 
  * Open
  * Save
  * SaveAs
  * Exit
* View
  *  Preview
  *  Export
* Other
  * Font
  * Find

If I press NVDA+alt+c, only hear "command gesture activated/command gesture deactivated".

Press alt+m, only hear "mark command window multi EX..", no thing appeared.

Press alt+l, only hear "multi EX command window short cut sub menu 1 of 9", no thing appeared.

Can any one help me?

**Comment by tsengwoody** [comment]
The menu that appear when you use the "Alt+M" and "Alt+L" commands are virtual menu. You can navigate and interact with the options using keyboard commands, and you will receive feedback through speech and Braille, but there won't be a visual menu displayed.
When you point/hear an item, you can press "Enter" to activate the corresponding action. For example, "Alt+L" is a LaTeX command, and it can insert the appropriate LaTeX command into the editing area, helping the writer quickly input LaTeX commands.
If it's a submenu, you can use the right arrow key or "Enter" to move to the next level, and the left arrow key to go back to the previous level. If you can't move further, there will be a beep to indicate that.

**Comment by tsengwoody** [comment]
Yes, we have also developed a sighted version of the editor with the same logical operations and LaTeX menu as the nonvisual editor. The primary target audience for this version is elementary and secondary school teachers, and it can create documents that are fully compatible with Access8Math.
For more information about the Access8Math editor and other detailed tutorials on accessibility for math in screen readers, we conducted related courses this year, and recordings are available on YouTube. You are welcome to refer to them.
Access8Math Web Editor: https://accessibility.twvip.org/Access8MathWeb
YouTube Tutorial: https://www.youtube.com/watch?v=E6DuuiuS6zo
